typescript/infra/config/environments/testnet3/igp.ts

```diff
@@ -19,23 +19,20 @@ function getGasOracles(local: TestnetChains) {
   );
 }
 
-export const igp: ChainMap<OverheadIgpConfig> = objMap(
-  owners,
-  (chain, owner) => {
-    return {
-      owner,
-      oracleKey: owner,
-      beneficiary: owner,
-      gasOracleType: getGasOracles(chain),
-      overhead: Object.fromEntries(
-        exclude(chain, supportedChainNames).map((remote) => [
-          remote,
-          multisigIsmVerificationCost(
-            defaultMultisigIsmConfigs[remote].threshold,
-            defaultMultisigIsmConfigs[remote].validators.length,
-          ),
-        ]),
-      ),
-    };
-  },
-);
+export const igp: ChainMap<IgpConfig> = objMap(owners, (chain, owner) => {
+  return {
+    owner,
+    oracleKey: owner,
+    beneficiary: owner,
+    gasOracleType: getGasOracles(chain),
+    overhead: Object.fromEntries(
+      exclude(chain, supportedChainNames).map((remote) => [
+        remote,
+        multisigIsmVerificationCost(
+          defaultMultisigIsmConfigs[remote].threshold,
+          defaultMultisigIsmConfigs[remote].validators.length,
+        ),
+      ]),
+    ),
+  };
+});
```

Comment:
ditto

Reply:
IgpConfig is shorter than OverheadIgpConfig